build.gradle

```diff
@@ -669,7 +674,105 @@ configure(updateBuildScript) {
     description = 'Updates the build script to the latest version'
 }
 
-// Deobfuscation
+// Parameter Deobfuscation
+
+task deobfParams {
+    doLast {
+        checkPropertyExists("channel")
+        checkPropertyExists("mappingsVersion")
+
+        String mcpDir = System.getProperty("user.home") + "/.gradle/caches/minecraft/de/oceanlabs/mcp/mcp_" + channel + "/" + mappingsVersion
+        String mcpZIP = mcpDir  + "/mcp_" + channel + "-" + mappingsVersion + "-" + minecraftVersion + ".zip"
```

Comment:
This could be more readable with string interpolation <https://groovy-lang.org/syntax.html#_string_interpolation>